charts/fission-core/values.yaml

```diff
@@ -110,11 +110,22 @@ router:
 
 ## Persist data to a persistent volume.
 persistence:
-  ## If true, fission will create/use a Persistent Volume Claim
+  ## If true (and storateType is not set), fission will create/use a Persistent Volume Claim
   ## If false, use emptyDir
   ##
   enabled: true
```

Comment:
I personally think being explicit about storage type as shown below is clearer to read and understand

```
persistence:
  enabled: true
  storageType: local | s3 | some_future_value (Default to local for backward compatibility)

  s3:
    accessKeyId: AHAHAAKHSKD
```

Reply:
This is done now. Please verify.